### PR TITLE
Update docs, bump cmake version

### PR DESCRIPTION
**Pull request by Leon0402**
Hi, 

I updated the docs and bumped the cmake version, which should fix the travis issue. How can I enable travis here, so I can see if the issue is actually resolved?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -756,32 +756,47 @@ For more details please check the project [wiki](https://github.com/fnc12/sqlite
...
 # Requirements
 
 * C++14 compatible compiler (not C++11 cause of templated lambdas in the lib).
-* libsqlite3 linked to your binary
+* Sqlite3 installed on your system and in the path, so cmake can find it (or linked to you project if you don't use cmake)
+
+## Support on Beerpay
+Hey dude! Help me out for a couple of :beers:!
+
+[![Beerpay](https://beerpay.io/fnc12/sqlite_orm/badge.svg?style=beer-square)](https://beerpay.io/fnc12/sqlite_orm)  [![Beerpay](https://beerpay.io/fnc12/sqlite_orm/make-wish.svg?style=flat-square)](https://beerpay.io/fnc12/sqlite_orm?focus=wish)
```
Comment (fnc12) [inline README.md:802]: please merge the latest commits - beerpay is removed. It is not required

**Comment by Leon0402** [comment]
@fnc12 Where can I see the failing travis build? Can you enable it here, so I can see the output? Otherwise it's hard to fix

**Comment by fnc12** [comment]
@Leon0402 https://travis-ci.org/github/fnc12/sqlite_orm/builds/714375736

File: README.md (hunk 1 above)
Comment (Leon0402) [reply README.md:802]: I actually am on the latest commit, I really don't know how that came back in, but I will just remove it

**Comment by Leon0402** [comment]
> @Leon0402 https://travis-ci.org/github/fnc12/sqlite_orm/builds/714375736

Why doesn't it show up here?

**Comment by Leon0402** [comment]
So I just had a look we need cmake 3.14 in order for this to work. But travis currently does only support cmake 3.12 ... ubuntu 20.04 is on the way for travis, but not currently available yet. 
So I will fix add a ppa for cmake later this day, hope that's fine.

**Comment by fnc12** [comment]
> Why doesn't it show up here?

Don't know. I need to investigate it.

**Comment by Leon0402** [comment]
Fixed, should be alright now!

**Comment by fnc12** [comment]
thank you

**Pull request merged**